src/pages/MissingPersonalDetails/substeps/Confirmation.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {View} from 'react-native';
+import Button from '@components/Button';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import SafeAreaConsumer from '@components/SafeAreaConsumer';
+import ScrollView from '@components/ScrollView';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useNetwork from '@hooks/useNetwork';
+import useThemeStyles from '@hooks/useThemeStyles';
+import type {CustomSubStepProps} from '@pages/MissingPersonalDetails/types';
+import CONST from '@src/CONST';
+import INPUT_IDS from '@src/types/form/PersonalDetailsForm';
+
+const PERSONAL_DETAILS_STEP_INDEXES = CONST.MISSING_PERSONAL_DETAILS_INDEXES.MAPPING;
+
+function Confirmation({personalDetailsValues: values, onNext, onMove}: CustomSubStepProps) {
```

Comment:
Should this be a confirmation step?
```suggestion
function ConfirmationStep({personalDetailsValues: values, onNext, onMove}: CustomSubStepProps) {
```